moto/ec2/models.py

```diff
@@ -2435,6 +2435,7 @@ def __init__(
         self.instance_tenancy = instance_tenancy
         self.is_default = "true" if is_default else "false"
         self.enable_dns_support = "true"
+        self.classic_link_enabled = "false"
```

Comment:
Strings and not booleans?

Reply:
I think this is just following in the style of the other attributes on this class.